wiki/Play_style/de.md

```diff
@@ -0,0 +1,173 @@
+
+### Tanzmatte
+
+Tanzmatten sind Controller für Spiele wie die Serien [Dance Dance Revolution](https://de.wikipedia.org/wiki/Dance_Dance_Revolution) (DDR) und [Pump It Up](https://en.wikipedia.org/wiki/Pump_It_Up_(video_game_series)) (PIU). ([StepMania](https://wikipedia.org/wiki/StepMania) ist ein halbwegs bekannter, von der Community betriebener Klon dieser beiden Spiele).
+
+Beim Spielen mit Tanzmatten "tanzen" Spieler auf der Matte, um Befehle einzugeben. DDR-Pads haben 4 Felder: hoch, runter, links und rechts. DDR-Solo-Pads fügen oben-links und oben-rechts hinzu. PIU-Pads have 5 Felder: oben-links, oben-rechts, unten-links, unten-rechts und in der Mitte. Es gibt auch einige 9 Pads, die sowohl mit PIU als auch mit DDR kompatibel sind, ohne dass ein Pad ausgetauscht werden muss. Normale DDR- und PIU-Pads werden oft im Doppelmodus in ihrem jeweiligen Spiel verwendet, wobei ein Spieler 2 Felder für einen Song benutzt. DDR-Solo-Spiele sind alle im Einzelspielermodus, daher sind sie in der Regel nicht verdoppelt.
+
+Ein wesentlicher Nachteil beim Benutzen einer Tanzmatte ist, dass es schwer sein kann, mehrere Tasten auf einmal zu drücken. Um 4 Noten auf einem DDR-Pad zu treffen, musst du entweder die Hände zusammen mit den Füßen einsetzen oder die Füße auf 2 Felder auf einmal platzieren.
+
+Noch ein Nachteil ist, dass osu!mania-Beatmaps möglicherweise ohne Rücksicht auf Abläufe von Tanzspielen entworfen wurden, wodurch die Menge an Karten limitiert wird, zu denen du (komfortabel) tanzen kannst. Tanzspiele werden oft mit Tanzkartenabläufen dargestellt. Die wichtigste Regel beim Erstellen von Karten für Tanzspiele wie DDR ist es, dass man bei jeder Note den Fuß wechseln kann (mit einigen Ausnahmen). Da manche osu!mania-Beatmaps nicht wirklich diesen Regeln folgen, kann es das Spielen dieser Beatmaps auf einer Tanzmatte sehr unangenehm machen.
+
+### Andere Stile
+
+- **Gitarren-/Schlagzeug-Set** (5/6K): Der Standardcontroller für das Gitarren-bezogene ([Guitar Hero](https://de.wikipedia.org/wiki/Guitar_Hero))/Schlagzeug-bezogene ([DrumMania](https://en.wikipedia.org/wiki/GuitarFreaks_and_DrumMania)) Rhythmusspiel. Spieler streamen auf ihrer Gitarre oder schlagen auf ihrem Schlagzeug und Zimbeln, um Befehle einzugeben. Schlagzeuger haben einen Nachteil im Vergleich zu Gitarrenspieler, weil sie ohne ein spezielles Fußpedal nur zwei Eingaben gleichzeitig machen können.
+- **Para Para Controller** (5K): Das ist der Standardcontroller für die Serie [Para Para Paradise](https://wikipedia.org/wiki/Para_Para_Paradise). Spieler bewegen ihre Arme, Ellbogen und Hände direkt über den Abstandssensoren, um Eingaben zu machen, basierend auf der Position der Hand und des Abstandssensors. Der Nachteil ist die Sensitivität des Eingabeempfängers und die physische Limitierung des Spielers.
```

Comment:
```suggestion
- **Para Para Controller** (5K): Das ist der Standardcontroller für die Serie [Para Para Paradise](https://wikipedia.org/wiki/Para_Para_Paradise). Spieler bewegen ihre Arme, Ellbogen und Hände direkt über den Abstandssensoren, um Eingaben zu machen, basierend auf der Position der Hand und die Abstandssensors. Der Nachteil ist die Sensitivität des Eingabeempfängers und die physische Limitierung des Spielers.
```

Reply:
What is the grammatical reasoning behind this change? Do you want me to write this:

> basierend auf der Position der Hand und den Abstandssensoren

Reply:
My bad, I must have misread that sentence. It's correct as is, ignore this :skull:

Reply:
No worries. I wanted to make sure that I understand you correctly 😄